groups/storage/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  common_tags = {
+    Environment = var.environment
+    Service     = var.service
+    Repository  = var.repository
```

Comment:
We'll need to agree on standards for this. We've just gone with `Terraform` as the provisioning repository elsewhere...

Reply:
Tags reworked in 3803c3f as discussed.